recipe/recipe.yaml

```diff
@@ -0,0 +1,75 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  version: "1.8.0"
+  tests_to_skip: not (test_http_404 or test_http_500 or test_http_502)
+  python_check_max: "3.13"
+  #   File "$SRC_DIR_env/lib/python3.14/site-packages/google/cloud/storage/blob.py", line 21, in <module>
+  #       import cgi
+  #   ModuleNotFoundError: No module named 'cgi'
+  # python_check_max: "3.14"
+
+package:
+  name: cached_path
+  version: ${{ version }}
+
+source:
+  url: https://github.com/allenai/cached_path/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 340d0a2a87b82450c810d62f3d293f263c8c555c0b6bfc2cec36d1e176730dc5
+
+build:
+  number: 1
+  noarch: python
+  script:
+    - ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --disable-pip-version-check
+
+requirements:
+  host:
+    - pip
+    - python ${{ python_min }}.*
+    - setuptools
+  run:
+    - boto3 >=1.0,<2.0
+    - filelock >=3.4,<4.0
+    - google-cloud-storage >=1.32,<3.0
+    - huggingface_hub >=0.8.1,<1.0
+    - packaging
+    - python >=${{ python_min }}
+    - requests
+    - rich >=12.1,<14.0
+
+tests:
+  - python:
+      imports: cached_path
+      pip_check: true
+      python_version:
+        - ${{ python_min }}.*
+        - ${{ python_check_max }}.*
+  - files:
+      source:
+        - test_fixtures/
+        - tests/
+    requirements:
+      run:
+        - flaky
+        - pytest
+        - python ${{ python_min }}.*
+        - responses
+    script:
+      # $SP_DIR is not set in rattler-build
```

Comment:
Can you still leave a comment that the upstream testing assumes a source-layout?